src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -278,7 +293,22 @@ public double getAngleSpeed() {
 		return launcherAngleEncoder.getVelocity();
 	}
 
-	public void setAngleManual(double joystickInput) {
+	public void restoreLimits() {
+		this.ignoreLimits = false;
+	}
+
+	public void setAngleManual(double joystickInput, boolean powerControl, boolean ignoreLimits) {
+		this.ignoreLimits = ignoreLimits;
+		if (powerControl || ignoreLimits) {
+			launcherAngleOneMotor.set(ignoreLimits ? joystickInput * 0.3 : joystickInput);
```

Comment:
Recommend bumping the multiplier to 0.5, so the motor has enough power to overcome a bent bellypan